### PR TITLE
Solve 5 in Elixir

### PR DESCRIPTION
**Pull request by lubien**
Intel® Core™ i3-5005U CPU @ 2.00GHz × 4
Exec time (s): 0.11

Named `solution_2.exs` since there's a PR with `solution_1.ex`

### CODE REVIEW

**Comment by ryukinix** [comment]
Approved

**Comment by rafaelcn** [comment]
Challenge accepted.

**Comment by leorog** [comment]
nice! :tada:

File: Problem005/Elixir/solution_2.exs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+defmodule Euler5 do
+  def solve(n) do
+    1..n |> Enum.reduce(&lcm/2)
+  end
+
+  def lcm(x, y), do: div(x * y, gcd(x, y))
+
+  def gcd(x, 0), do: x
+  def gcd(x, y) do
+    gcd(y, rem(x, y))
```
Comment (ryukinix) [inline Problem005/Elixir/solution_2.exs]: delicious

**Pull request merged**